cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -723,6 +723,8 @@
     // The following properties can be defined in cdap-site and overridden in individual pipeline config xml
     public static final String PIPELINE_BUFFER_SIZE = "log.process.pipeline.buffer.size";
     public static final String PIPELINE_EVENT_DELAY_MS = "log.process.pipeline.event.delay.ms";
+    public static final String PIPELINE_EVENT_OUT_OF_ORDER_MS = "log.process.pipeline.event.out.of.order.ms";
```

Comment:
This seems not applicable anymore. What we want is just a event delay so that we can have a upper bound on the event time while doing the search.